pymemcache/pool.py

```diff
@@ -43,7 +46,7 @@ def __init__(
             raise ValueError('"max_size" must be a positive integer')
         self.max_size = max_size
         self.idle_timeout = idle_timeout
-        self._idle_clock = time.time if idle_timeout else int
+        self._idle_clock = time.time if idle_timeout else lambda: 0
```

Comment:
Would `float` work here instead?

Reply:
mypy throws the following error: `pymemcache/pool.py:113: error: "object" not callable`

Reply:
Ah. This might need to be:

```suggestion
        self._idle_clock: Callable[[], float] = time.time if idle_timeout else float
```

Reply:
```pymemcache/pool.py:49: error: Incompatible types in assignment (expression has type "object", variable has type "Callable[[], float]")```

```python
from typing import Callable
import time

a: Callable = int
b: Callable = time.time
c: Callable = time.time
idle_timeout = False
d: Callable = time.time if idle_timeout else float
```
```
$ mypy foo.py
foo.py:8: error: Incompatible types in assignment (expression has type "object", variable has type "Callable[..., Any]")
Found 1 error in 1 file (checked 1 source file)```

Looks like the issue is the if statement will refactor